crates/krun/src/bin/krun.rs

```diff
@@ -294,90 +310,65 @@ fn main() -> Result<()> {
     let krun_guest_path = find_krun_exec("krun-guest")?;
     let krun_server_path = find_krun_exec("krun-server")?;
 
-    let mut krun_guest_args: Vec<CString> = vec![
-        CString::new(username).expect("username should not contain NUL character"),
-        CString::new(format!("{}", getuid().as_raw()))
-            .expect("uid should not contain NUL character"),
-        CString::new(format!("{}", getgid().as_raw()))
-            .expect("gid should not contain NUL character"),
+    let mut krun_guest_args: Vec<String> = vec![
+        krun_guest_path,
+        username,
+        format!("{}", getuid().as_raw()),
+        format!("{}", getgid().as_raw()),
```

Comment:
I think our MSRV allows this:

```suggestion
        format!("{uid}", uid = getuid().as_raw()),
        format!("{gid}", gid = getgid().as_raw()),
```

Reply:
Applied, thanks.